lib/libcompiler_rt/Makefile

```diff
@@ -19,9 +19,6 @@ MK_WERROR.gcc=	no
 
 .include "Makefile.inc"
 
-# subrepo-cheri-compiler-rt doesn't have lse.S yet
-.if ${CRTSRC:M*subrepo*} == ""
-
 # Out-of-line LSE atomics helpers for aarch64
 .if ${MACHINE_CPUARCH} == "aarch64"
```

Comment:
I don't think this will build for purecap?

Reply:
Is LSE one of the things Morello assumes is always present thus not requiring out-of-line atomics or can the compiler still use these routines on Morello?

Reply:
lse.S has a hard-coded .arch which may be why it builds, but it's probably not "correct"

```
#ifdef HAS_ASM_LSE
.arch armv8-a+lse
#else
.arch armv8-a
#endif
```

Reply:
There aren't alternate base versions of any atomics, even load/store exclusive, so it shouldn't build... though it looks like if HAS_ASM_LSE isn't defined it'll use hard-coded encodings, so that's probably why it works.

And yes, LSE was introduced as a mandatory extension in Armv8.1-A, so present in Morello; the outlined atomics only exist as an I-can't-believe-it's-not-an-IFUNC way of using them when available in order to support Armv8(.0)-A.

Reply:
Oh it's not even using C64... ouch

Reply:
Yeah, I can just disable them for purecap then.